### DISCUSSION TITLE
How to make fraction's content bigger?

### DISCUSSION THREAD

**Discussion by RafailFridman** [typst/typst#1116] [Q&A] [answered]
Hi, I have this compound fraction:
`(sum_i (x_i mu_i)/sigma_i^2)/(sum_i (mu_i^2)/sigma_i^2)`
It is currently being rendered as 
![image](https://user-images.githubusercontent.com/22198039/236466616-2bc55424-69b1-4ad3-adef-068fb4467018.png)

While in latex, with the use of `\dfrac` I can get this:
![image](https://user-images.githubusercontent.com/22198039/236466687-14d810c9-bca6-4cba-9665-35dbeaeffac1.png)

Notice how in latex the content in the inside fractions has a larger relative size wrt the fraction itself than in typst.
Is there a way to enlarge it?

Thanks

**Comment by Enivex** [comment]
Try
```
$ #box[$ sum_i (x_i mu_i)/(sigma_i^2) $]/(#box[$ sum_i (mu_i^2)/sigma_i^2 $]) $
```

(If you really want to you can move the sum symbols out of the box, but then they look way too small, like in your latex example)

**Reply by RafailFridman** [reply]
Thank you!
In this case it works, but it does not seem to help with a more complicated fraction:
`$ ((2 S^2)/(V^2 sum_i (mu_i^2)/(sigma_i^2)) mu_i V^2)/(2 S^2 sigma_i^2)  $`
![image](https://user-images.githubusercontent.com/22198039/236475970-44f6f65f-cdb1-4032-8d4d-bb3141d2a080.png)

`$ #box[$ ((2 S^2)/(V^2 sum_i (mu_i^2)/(sigma_i^2)) mu_i V^2) $]/#box[$ (2 S^2 sigma_i^2) $] $`
![image](https://user-images.githubusercontent.com/22198039/236476128-87d44fe3-cdfc-4d7c-b3e1-01b538f40b42.png)

**Reply by Enivex** [reply]
That's likely because you gave it an equation number.

I'm not sure what the intended behavior is. You could consider opening an issue.

**Reply by Enivex** [reply, marked as answer]
Here's an implementation of dfrac.

```
#set math.equation(numbering: "(1)")
#let nonumeq = math.equation.with(block: true, numbering: none)
#let dm(x) = box[#nonumeq[#x]]
#let dfrac(x,y) = math.frac(dm(x),dm(y))
$ dfrac( frac(2 S^2,V^2 sum_i (mu_i^2)/(sigma_i^2)) mu_i V^2 , 2 S^2 sigma_i^2) $
```
![image](https://user-images.githubusercontent.com/2191754/236489520-0825147d-a3c5-459a-8376-003590aee9f0.png)

If you also want the inner fraction to be display math, then you just change the frac to a dfrac and get

![image](https://user-images.githubusercontent.com/2191754/236489642-6f878767-dcf3-407f-a028-d87ac2f4a9d9.png)

**Reply by RafailFridman** [reply]
Thank you!

**Comment by Qazalbash** [comment]
How to enable this behaviour globally?
